src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/Api/MainHandlers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using $safeprojectname$.ViewModels;
+using Starcounter;
+
+namespace $safeprojectname$.Api
+{
+    internal static class MainHandlers
+    {
+        public static void Register()
+        {
+            Application.Current.Use(new HtmlFromJsonProvider());
+            Application.Current.Use(new PartialToStandaloneHtmlProvider());
+
+            Handle.GET("/$safeprojectname$", () =>
+            {
+                var page = new HelloStarcounterPage
+                {
+                    Visitors = {Data = Db.SQL<GuestBookEntry>("SELECT g FROM GuestBookEntry g")}
+                };
+
+                if (Session.Current == null)
+                {
+                    Session.Current = new Session(SessionOptions.PatchVersioning);
+                }
+
+                page.Session = Session.Current;
+
+                return page;
```

Comment:
Since this template should only contain the structure of the app like folders and such most of the code in the HelloWorld app would be useless and removed any way bu the developer.

With this the developer can still augment and build upon the existing code to get a little head start from the blank template we have.